### PR TITLE
Use combined IP/IP2 library

### LINKED ISSUE #242: Test UFS_UTILS with combined IP/IP2 library
@kgerheiser is combining IP and IP2 into a single library.  Test this library with UFS_UTILS.

https://github.com/NOAA-EMC/NCEPLIBS-ip2/pull/29

### PR DESCRIPTION
**Pull request by GeorgeGayno-NOAA**
## DESCRIPTION OF CHANGES: 
The IP and IP2 libraries were combined as IPLIB v4.0.0. Update UFS_UTILS to use this version. 

Note: the UFS_UTILS codes were modified so they are backwards compatible with IPLIB v3.

## TESTS CONDUCTED: 
All consistency tests passed. See below for details.

## DEPENDENCIES:
None.

## DOCUMENTATION:
N/A

## ISSUE: 
Fixes #242.

### CODE REVIEW

**Comment by GeorgeGayno-NOAA** [comment]
On Hera, 5cf0dd8 was compiled, and the `chgres_cube`, `global_cycle`, `grid_gen` and `snow2mdl` regression tests were successfully run.

**Comment by edwardhartnett** [comment]
Are there some tests to go along with these changes?

**Comment by GeorgeGayno-NOAA** [comment]
The branch at b2696d9 was compiled on Jet. All consistency tests passed.

**Comment by GeorgeGayno-NOAA** [comment]
The branch at f426727 was compiled on Orion, Jet, Hera and Cactus. All consistency tests passed.

**Comment by GeorgeGayno-NOAA** [comment]
@LarissaReames-NOAA @JeffBeck-NOAA This is ready to merge. It just needs your approval.

File: modulefiles/build.wcoss2.intel.lua
Comment (JeffBeck-NOAA) [inline modulefiles/build.wcoss2.intel.lua:55]: @GeorgeGayno-NOAA, so the new version of the IP library was built in your own user space?  Is there a general user space on WCOSS2 where it could go instead?

Comment (GeorgeGayno-NOAA) [reply modulefiles/build.wcoss2.intel.lua:55]: That was a comment that should have been removed. 

I also removed the specification of the MODULEPATH below that was required to find the new library.  I see that is no longer needed. (See a682ce8). These machines are always in a state of flux.

Thanks for your careful review.

**Comment by GeorgeGayno-NOAA** [comment]
The branch at a682ce8 was compiled on Dogwood. The `chgres_cube`, `global_cycle`, `grid_gen` and `snow2mdl` consistency tests all passed as expected.

**Comment by GeorgeGayno-NOAA** [comment]
@edwardhartnett All our Github actions are now breaking. The log files indicate a timeout during the download of the bufr library's static files. UFS_UTILS does not use bufr lib. (I also see the CRTM is being compiled. UFS_UTILS does not use CRTM either.) Is there a quick fix for this?

File: modulefiles/build.wcoss2.intel.lua
Comment (JeffBeck-NOAA) [reply modulefiles/build.wcoss2.intel.lua:55]: Oh, yes, sorry.  I missed that this was a comment!   Thanks for making these changes.

**Review by JeffBeck-NOAA — APPROVED** [review]

**Comment by GeorgeGayno-NOAA** [comment]
After merging the latest updates from 'develop' (0f50253), I reran the `global_cycle` consistency tests on Hera. They all passed.

Will merge this PR.

**Pull request merged**
